src/components/OverviewSelectors.jsx

```diff
@@ -3,43 +3,50 @@ import React from 'react';
 const OverviewSelectors = ({ pickSize, sizes, currentSize, quantities, addToCart, pickQuantity }) => {
   if (quantities.length > 0) {
     return (
-      <div className="overview-selectors"> Size:
-        <select className="overview-dropdown" onChange={(e) => { pickSize(e) }}>
+      <div>
+      <div className="overview-selectors">
+      <div></div>
+        <select data-testid="size-selector" className="overview-dropdown" onChange={(e) => { pickSize(e) }}>
           <option value={`${currentSize}`}> {currentSize} </option>
           {sizes.map((size) => (
             <option value={size}>{size}</option>
           ))}
         </select>
-        Quantity:
+        <div></div>
         <select className="overview-dropdown" onChange={(e) => { pickQuantity(e) }}>
           <option value="1" default> 1 </option>
           {quantities.slice(1).map((quantity) => (
             <option value={quantity}>{quantity}</option>
           ))}
         </select>
-        <div>
-          <button className="cartButton" onClick={() => { addToCart() }}>Add to Cart</button>
+      </div>
+        <div className="overview-buttons">
+          <button data-testid="cartbutton" className="cartButton" onClick={() => { addToCart() }}>Add to Cart</button>
+          <button className="starButton"><i class="fa-solid fa-star"></i></button>
         </div>
       </div>
     )
   }
   else {
     return (
+      <div>
```

Comment:
Might be best to give some of these divs a class/id even if you are not styling them. Just helps with readability but not super important.